backend-tests/tests/test_rbac_v2.py

```diff
@@ -439,19 +552,9 @@ def test_deploy_to_group(self, clean_mongo, test_case):
                 "user": {
                     "name": "test1-UUID@example.com",
                     "pwd": "password",
-                    "roles": ["test"],
+                    "roles": ["RBAC_ROLE_PERMIT_ALL"],
```

Comment:
would be better to test it with test role containing "ConfigureDevices" permission

Reply:
Another test cases contains `ConfigureDevices` permission, in line 628